apis/core/v1alpha1/resource.go

```diff
@@ -167,15 +171,31 @@ type ProviderSpec struct {
 	CredentialsSecretRef *SecretKeySelector `json:"credentialsSecretRef,omitempty"`
 }
 
-// A ProviderConfigSpec defines the common way to get to the necessary objects to connect
-// to the provider.
+// A ProviderConfigSpec defines the common way to get to the necessary objects
+// to connect to the provider.
 type ProviderConfigSpec struct {
 	// CredentialsSecretRef references a specific secret's key that contains
 	// the credentials that are used to connect to the provider.
 	// +optional
 	CredentialsSecretRef *SecretKeySelector `json:"credentialsSecretRef,omitempty"`
 }
 
+// A ProviderConfigStatus defines the observed status of a ProviderConfig.
+type ProviderConfigStatus struct {
+	// Users of this provider configuration.
+	Users *int64 `json:"users,omitempty"`
```

Comment:
Does this need to be a pointer type? I think we have typically not used pointers in status. It seems like having the zero-value of `int64` (0) when not set would be fine here